Packs/CommonScripts/Scripts/ExportAuditLogsToFile/ExportAuditLogsToFile.py

```diff
@@ -6,43 +6,94 @@
 from datetime import date, timedelta
 
 
+def get_audit_logs(res: Dict):
+
+    def get_xsoar6_audit_logs():
+        return res.get('audits') or []
+
+    def get_xsoar8_audit_logs():
+        return (res.get('reply') or {}).get('data') or []
+
+    return get_xsoar6_audit_logs() or get_xsoar8_audit_logs()
+
+
+def get_audit_logs_count(res: Dict):
+    def get_xsoar6_audit_logs_count():
+        return res.get('total', 0)
+
+    def get_xsoar8_audit_logs_count():
+        return res.get('total_count', 0)
```

Comment:
```suggestion
    def get_xsoar6_audit_logs_count():
        return res.get('total') or 0

    def get_xsoar8_audit_logs_count():
        return res.get('total_count') or 0
```
This is better in case the field has an empty value, then it will take the empty value rather than 0